templates/kubeflow/index.html

```diff
@@ -6,16 +6,18 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1mJq8PxH8FVRVF9i1Xzh0f6M73eyjqZE4NaKYHRcRAKM/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip is-deep is-bordered">
-  <div class="row">
-    <div class="col-7">
-      <h1>AI / ML from workstation to production</h1>
-      <p>Ubuntu is the platform to power your Artificial Intelligence ambitions &mdash; from developer workstations, to racks, to clouds and to the edge with smart connected IoT.</p>
-      <p>Canonical&rsquo;s Kubeflow supports the most popular tools for machine learning &mdash; starting with JupyterHub and Tensorflow &mdash; in a standardised workflow running on Kubernetes.</p>
-      <p>
-        <a class="p-button--positive" href="/kubeflow/install">Get started by installing Kubeflow</a>
-        <a class="p-button--neutral js-invoke-modal" href="/kubeflow/contact-us?product=ai-index">Call the experts</a>
-      </p>
+<section class="p-strip is-deep is-bordered u-no-padding--top u-no-padding--bottom">
```

Comment:
You can add the p-strip--suru-topped to line 9 and remove p-strip u-no-padding--top u-no-padding--bottom. And also you can remove the next level div (line 10)